external/anomaly/tests/reference/ote_anomaly_classification_padim/nncf/nncf_quantization.dot

```diff
@@ -0,0 +1,240 @@
+strict digraph  {
```

Comment:
Why are these dot files added? Can't we regenerate these from the tests?

Reply:
No. The idea of test to compare graph of model with reference that generated when it works correctly. If graph changes without update nncf configs, it will be unwanted behavior. It's like reference evaluation metric, you want to compare current state with reference.

Reply:
Okay that makes sense. But it is something for us to keep in mind when we make any changes to the model.